assets/projects.css

```diff
@@ -0,0 +1,40 @@
+.project-filter {
```

Comment:
same, bs classes pls

Reply:
reopen for `border`, `background-color` and `color`

Reply:
Sorry, thought these were css variables and got scared to touch it last time. Moved them to their tags in list.html